seedscript.sql

```diff
@@ -0,0 +1,64 @@
+DROP DATABASE IF EXISTS droptable;
+
+CREATE DATABASE droptable;
+
+USE droptable;
+
+CREATE TABLE restaurants (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(50) NOT NULL,
+  overallAvg DECIMAL(4, 3) DEFAULT NULL,
+  foodAvg DECIMAL(4, 3) DEFAULT NULL,
+  serviceAvg DECIMAL(4, 3) DEFAULT NULL,
+  ambienceAvg DECIMAL(4, 3) DEFAULT NULL,
+  valueAvg DECIMAL(4, 3) DEFAULT NULL,
+  wyrAvg DECIMAL(5, 2) DEFAULT NULL,
+  numReviews INT DEFAULT 0,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(50) NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE reviews (
+  id INT NOT NULL AUTO_INCREMENT,
+  user INT NOT NULL,
+  restaurant INT NOT NULL,
+  overall TINYINT(1) NOT NULL,
+  food TINYINT(1) NOT NULL,
+  service TINYINT(1) NOT NULL,
+  ambience TINYINT(1) NOT NULL,
+  value TINYINT(1) NOT NULL,
+  wyr BIT(1) NOT NULL,
+  text VARCHAR(4000) NOT NULL,
+  date TIMESTAMP NOT NULL,
+  partySize TINYINT NOT NULL,
+  PRIMARY KEY (id),
```

Comment:
the keywords PRIMARY KEY can be written on the end of line 27 to accomplish the same effect, but this is fine if you like it better this way.